terraform/standalone/main.tf

```diff
@@ -20,4 +31,39 @@ module "dependencies" {
   vpc_id             = module.vpc.inner.vpc_id
   subnet_ids         = module.vpc.inner.private_subnets
   availability_zones = module.vpc.inner.azs
-}
\ No newline at end of file
+}
+
+module "retail_app_eks" {
+  source = "../lib/eks"
+
+  environment_name = var.environment_name
+  cluster_version  = "1.24"
+  vpc_id           = module.vpc.inner.vpc_id
+  subnet_ids       = module.vpc.inner.private_subnets
+  tags             = module.tags.result
+}
+
+# Setup deploying role cluster with auth
+resource "null_resource" "setup_cluster_access" {
```

Comment:
Can we use this method for setting up cluster auth locally? https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/vpc-cni-custom-networking/main.tf#L140

Reply:
cluster auth have been updated